roles/docker_traefik/defaults/main.yml

```diff
@@ -32,3 +32,9 @@ traefik__network: public
 traefik__refresh: 15
 
 traefik__state: started
+
+
+use_nfs_server: false
```

Comment:
Order all variables alphabetically.
All variables must be prefixed with `traefik__`
`use_nfs_server` should be renamed to `traefik__use_nfs`